.github/workflows/ci.yml

```diff
@@ -3,11 +3,15 @@ name: CI
 on:
   workflow_dispatch: {}
   push:
-    branches: [master]
+    branches:
```

Comment:
Weird that this got pulled into diff

Reply:
It's not, every time we update beta branch codegen we also merge the master in.

Reply:
This way we get latest fixes from master in case they are needed for newly generated code.

Reply:
makes sense